src/web/app/pages-help/instructor-help-page/instructor-help-general-section/instructor-help-general-section.component.specs.ts

```diff
@@ -0,0 +1,31 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { InstructorHelpPanelComponent } from '../instructor-help-panel/instructor-help-panel.component';
+import { InstructorHelpGeneralSectionComponent } from './instructor-help-general-section.component';
+import { TeammatesRouterModule } from '../../../components/teammates-router/teammates-router.module';
+import { RouterTestingModule } from '@angular/router/testing';
+
+describe('InstructorHelpGeneralSectionComponent', () => {
+    let component: InstructorHelpGeneralSectionComponent;
+    let fixture: ComponentFixture<InstructorHelpGeneralSectionComponent>;
+  
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        declarations: [
+          InstructorHelpGeneralSectionComponent,
+          InstructorHelpPanelComponent,
+        ],
+        imports: [RouterTestingModule, TeammatesRouterModule],
+      })
+      .compileComponents();
+    }));
+  
+    beforeEach(() => {
+      fixture = TestBed.createComponent(InstructorHelpGeneralSectionComponent);
+      component = fixture.componentInstance;
+      fixture.detectChanges();
+    });
+  
+    it('should create', () => {
+      expect(component).toBeTruthy();
+    });
+  });
```

Comment:
Line break overhere?

Reply:
The screenshots show the latest but not all the changes as the questions were spread out through some of the sections so I showed some examples, but I can add screenshots of all of them if wanted. I will add the line break as well.